src/commands/synthetics/mobile.ts

```diff
@@ -1,34 +1,41 @@
 import * as crypto from 'crypto'
 import fs from 'fs'
 
-import {APIHelper} from './api'
-import {Test, TestPayload, UserConfigOverride} from './interfaces'
+import {APIHelper, EndpointError, formatBackendErrors} from './api'
+import {PresignedUrlResponse, Test, TestPayload, UserConfigOverride} from './interfaces'
 
-export const getMD5HashFromFile = async (file: string): Promise<string> => {
+export const getFileAndMD5HashFromFile = async (filePath: string): Promise<{appSize: number; md5: string}> => {
```

Comment:
Shouldn't this function be called something like `getSizeAndMD5HashFromFile` as you get the size of the file and not the file itself ?

Reply:
I completely miss it, completely agree 👍